jujugui/static/gui/src/app/components/deployment-bar/deployment-bar.js

```diff
@@ -0,0 +1,45 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('deployment-bar', function() {
+
+  juju.components.DeploymentBar = React.createClass({
+
+    render: function() {
+      var changeCount = Object.keys(this.props.currentChangeSet).length;
+      return (
+        <juju.components.Panel
+          instanceName="deployment-bar-panel"
+          visible={true}>
+          <juju.components.DeploymentBarChangeCount
+            count={changeCount} />
+          <juju.components.DeploymentBarDeployButton
```

Comment:
If you changed to the inspector button component as I commented above you'd have to move some of that logic into this component...I think that would be a reasonable move.